suite-native/feature-flags/src/featureFlagsSlice.ts

```diff
@@ -34,7 +34,8 @@ export const featureFlagsInitialState: FeatureFlagsState = {
         process.env.EXPO_PUBLIC_FF_IS_CONNECT_POPUP_ENABLED === 'true',
     [FeatureFlag.IsDebugKeysAllowed]: process.env.EXPO_PUBLIC_FF_IS_DEBUG_KEYS_ALLOWED === 'true',
     [FeatureFlag.IsWalletConnectEnabled]:
-        process.env.EXPO_PUBLIC_FF_IS_WALLET_CONNECT_ENABLED === 'true',
+        process.env.EXPO_PUBLIC_FF_IS_WALLET_CONNECT_ENABLED === 'true' ||
+        (isAndroid() && process.env.EXPO_PUBLIC_FF_IS_WALLET_CONNECT_ENABLED !== 'false'),
```

Comment:
Imo this says:
if ff enabled or on android if ff not disabled. which is the same. or no?


```suggestion
        isAndroid() || process.env.EXPO_PUBLIC_FF_IS_WALLET_CONNECT_ENABLED === 'true',
```

Reply:
Also afaik FFs are persisted, so users who previously installed app will have FF set to false and there would be migration needed.

Reply:
It's the same as the flag for device connection. So Android - on by default, iOS - off by default. 
Good point about migration, but maybe to make things simple we could change the key of the feature flag to reset it. (eg. `IsWalletConnectEnabled: 'isWalletConnectEnabled'` -> `'isWalletConnectEnabled_v2'`)

Reply:
🤔  Good point about that device connect, I'll have to inspect more why it is defined like that, so leave it as is 👍 

Renaming is simple workaround 👌.

Reply:
Added fixups